css/style.css

```diff
@@ -1,19 +1,168 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+:root {
+  --grey-dark: #292b2c;
+  --grey-mid: #808080;
+  --grey-light: #e4e4e4;
+  --orange-dark: #c05326;
+  --orange-light: #f7eae4;
+  --white: #fff;
+}
```

Comment:
Good use of css variables (even though we haven't learnt them yet!), this helps to make your code really clean 🙌 

Did you find it useful using them?

Reply:
I totally love it. I first encounter it during "Intro to Digital Course". It is maybe more typing at the beginning but when code gets longer and longer, it make my life easier.